### PR TITLE
Preparing release version 3.2.2

### PR DESCRIPTION
**Pull request by nicoddemus**
- [Travis](https://travis-ci.org/nicoddemus/devpi-cloud-test-pytest/builds/272684020)
- [AppVeyor](https://ci.appveyor.com/project/nicoddemus/devpi-cloud-test-pytest/build/1.0.22)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,6 +8,53 @@
...
+  ``pytest.mark.MARKER_NAME.__call__`` (`#2604
+  <https://github.com/pytest-dev/pytest/issues/2604>`_)
+
+- In one of the simple examples, use `pytest_collection_modifyitems()` to skip
+  tests based on a command-line option, allowing its sharing while preventing a
+  user error when acessing `pytest.config` before the argument parsing. (`#2653
+  <https://github.com/pytest-dev/pytest/issues/2653>`_)
+
+
+Trivial/Internal Changes
```
Comment (tgoodlet) [inline CHANGELOG.rst]: Huh weird I would have expected the `pluggy` de-vendoring to show up no?

Comment (nicoddemus) [reply CHANGELOG.rst]: That will be in `3.3.0` only, when we release from `features`. 😉

Comment (nicoddemus) [reply CHANGELOG.rst]: We don't want to suddenly de-vendor dependencies in bugfix releases,

Comment (tgoodlet) [reply CHANGELOG.rst]: 🤦‍♂️

**Pull request merged**